www/utils/supabase/queries.ts

```diff
@@ -32,7 +32,6 @@ export const getSubscription = cache(async (supabase: SupabaseClient) => {
   const { data: subscription, error } = await supabase
     .from('subscriptions')
     .select('*, prices(*, products(*))')
-    .in('status', ['trialing', 'active'])
```

Comment:
@bLopata Calling out this change. Did this so it would also query cancelled subscriptions. Otherwise it'll try to make a new trial subscription if they don't have one.